xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AccountTransactionsIT.java

```diff
@@ -104,8 +104,6 @@ public void listTransactionWithIndexRange() {
         .build()
     );
 
-    assertThat(results.ledgerIndexMinimum()).isEqualTo(minLedger);
-    assertThat(results.ledgerIndexMaximum()).isEqualTo(maxLedger);
```

Comment:
I don't think we should remove these checks. Instead, I think we should either (1) Only talk to https://s2-clio.ripple.com or (2) Somehow figure out how to only talk to a reporting server, or (3) Get xrplcluster.com to support JSON-RPC and talk to that.

Overall, #1 is the future, but CLIO seems to be giving incorrect values, so I'm hesitant to use it. Simply removing these checks doesn't seem like the right path either though since these checks are serving a purpose.